src/handlers/DibiHandler.php

```diff
@@ -110,12 +111,14 @@ public function getInfo()
 			return NULL;
 		}
 
-		$query = 'EXPLAIN (FORMAT JSON) ' . $this->event->sql;
-		$explain = $this->event->connection->nativeQuery($query);
-		$data = json_decode($explain->fetchSingle(), TRUE);
-		return Html::el()->setHtml(Dumper::toHtml($data, [
-			Dumper::COLLAPSE => TRUE,
-		]));
+		if(\Nette\Utils\Strings::startsWith($this->event->sql, 'SELECT')) {
```

Comment:
that's true, I needed this quick up and running, sadly for version older then 5.6.3 and dibi 3 so I did not bother much, works good for most cases that I'm using so it's good enough for me. Basically I shouldn't create pull request for that since it's mostly for me...